coalahtml/_coalahtml/app/app.js

```diff
@@ -2,12 +2,35 @@
 
 angular
   .module('coalaHtmlApp', ['ngRoute', 'ngSanitize'])
-  .run(function($rootScope) {
+  .run(function($rootScope, $http) {
     $rootScope.LOG_LEVELS = ["DEBUG", "INFO", "WARNING", "ERROR"];
     $rootScope.RESULT_SEVERITY = ["INFO", "NORMAL", "MAJOR"];
     $rootScope.SEVERITY_TO_BOOTSTRAP = ["info", "warning", "danger"];
     $rootScope.LOG_LEVEL_TO_BOOTSTRAP = ["primary", "info", "warning",
                                          "danger"];
+    var getData = function (url) {
+      return $http.get(url);
+    };
```

Comment:
Oh nvm. My bad. I misinterpreted your comment.
ᐧ

On Fri, Jun 3, 2016 at 3:10 PM, Tushar Gautam <tushar.rishav@gmail.com>
wrote:

> I followed http://stackoverflow.com/a/29849524/3673031
> ᐧ
>
> On Fri, Jun 3, 2016 at 3:07 PM, Tushar Gautam <tushar.rishav@gmail.com>
> wrote:
>
>> I needed to have those data files loaded, before I could actually use
>> them later. So I made it synchronous.
>> ᐧ
>>
>> On Fri, Jun 3, 2016 at 3:06 PM, Tushar Gautam <tushar.rishav@gmail.com>
>> wrote:
>>
>>> So this is synchronous. It returns a promise actually.
>>> ᐧ
>>>
>>> On Fri, Jun 3, 2016 at 2:55 PM, Attila <notifications@github.com> wrote:
>>>
>>>> In coalahtml/_coalahtml/app/app.js
>>>> <https://github.com/coala-analyzer/coala-html/pull/52#discussion_r65679661>
>>>> :
>>>>
>>>> >      $rootScope.LOG_LEVELS = ["DEBUG", "INFO", "WARNING", "ERROR"];
>>>> >      $rootScope.RESULT_SEVERITY = ["INFO", "NORMAL", "MAJOR"];
>>>> >      $rootScope.SEVERITY_TO_BOOTSTRAP = ["info", "warning", "danger"];
>>>> >      $rootScope.LOG_LEVEL_TO_BOOTSTRAP = ["primary", "info", "warning",
>>>> >                                           "danger"];
>>>> > +    var getData = function (url) {
>>>> > +      return $http.get(url);
>>>> > +    };
>>>>
>>>> How is this better that raw $http.get(url)?
>>>>
>>>> —
>>>> You are receiving this because you authored the thread.
>>>> Reply to this email directly, view it on GitHub
>>>> <https://github.com/coala-analyzer/coala-html/pull/52/files/b7f706d2171c77e341e20b5b680c0c87b1b38b7e#r65679661>,
>>>> or mute the thread
>>>> <https://github.com/notifications/unsubscribe/AHDgOYIINWsziJFcvJgu0GgE60gQeRhgks5qH_L9gaJpZM4IslPq>
>>>> .
>>>>
>>>
>>>
>>>
>>> --
>>> Best regards,
>>> Tushar Gautam
>>> http://gtushar.co
>>>
>>
>>
>>
>> --
>> Best regards,
>> Tushar Gautam
>> http://gtushar.co
>>
>
>
>
> --
> Best regards,
> Tushar Gautam
> http://gtushar.co
>



-- 
Best regards,
Tushar Gautam
http://gtushar.co